eventlog/service.go

```diff
@@ -143,12 +148,11 @@ filter:
 	return reply, nil
 }
 
-func decodeAndCheckEvent(coll byzcoin.ReadOnlyStateTrie, eventBuf []byte) (*Event, error) {
-	// Check the timestamp of the event: it should never be in the future,
-	// and it should not be more than 30 seconds in the past. (Why 30 sec
-	// and not something more auto-scaling like blockInterval * 30?
-	// Because a # of blocks limit is too fragile when using fast blocks for
-	// tests.)
+func decodeAndCheckEvent(rst byzcoin.ReadOnlyStateTrie, eventBuf []byte) (*Event, error) {
+	// Check the timestamp of the event: it should never be in the future beyond the current block,
```

Comment:
you accept `4 * config.BlockInterval`, which is fine, but doesn't match your comment here.